sw/device/silicon_creator/rom/rom_start.S

```diff
@@ -260,6 +260,15 @@ LABEL_FOR_TEST(kRomStartWatchdogEnabled)
   li   t0, SW_STRAP_2_INSEL
   sw   t0, (SW_STRAP_2_PERIPH * 4)(a0)
 
+  // Spin for a minimum of 5us worth of cycles (assuming 100MHz clock), while we
+  // wait for the RMA strap pull enables to propagate to the physical pads.
+  // Since this is a wait loop, disable the watchdog.
+  csrw mcycle, zero
+  li   t0, 500  // 500 clock cycles with 100MHz clock is 5us.
```

Comment:
nit: this could be defined in `sw/device/silicon_creator/lib/base/chip.h` as a macro so that we have a single source of truth for asm and c.

Reply:
Fixed in #18020